app/assets/stylesheets/avalon.scss

```diff
@@ -38,6 +38,11 @@ body {
   }
 }
 
+/* Override Bootstrap CSS */
+.alert {
+  margin-bottom: 10px;
+}
+
```

Comment:
Does this negatively impact any other pages with alert messages?

Reply:
I checked this in other pages and seems to not have any negative effect.